src/main/java/sx/blah/discord/util/RequestBuffer.java

```diff
@@ -56,7 +56,11 @@ public static void request(IVoidRequest request) {
 	 * @return The number of incomplete requests.
 	 */
 	public static int getIncompleteRequestCount() {
-		return requests.size();
+		final int[] count = {0};
```

Comment:
You should use an `AtomicInteger` as this is the use case for such a class as well as it being more consistent with the rest of the source.